src/js/modal-dialog.js

```diff
@@ -461,19 +461,7 @@ class ModalDialog extends Component {
     }
 
     const focusableEls = this.focusableEls_();
-    const activeEl = this.el_.querySelector(':focus');
-    let focusIndex;
-
-    for (let i = 0; i < focusableEls.length; i++) {
-      if (activeEl === focusableEls[i]) {
-        focusIndex = i;
-        break;
-      }
-    }
-
-    if (document.activeElement === this.el_) {
-      focusIndex = 0;
-    }
+    const focusIndex = focusableEls.indexOf(document.activeElement);
```

Comment:
`indexOf` isn't supported on a `NodeList` in ie 11, so we won't be able to use it

Reply:
I'd also be wary of making a change here.
This was based on https://github.com/gdkraus/accessible-modal-dialog/blob/master/modal-window.js#L95-L109 and doing a change here would require testing across a bunch of browsers, and we don't really have the bandwidth to do that.

Reply:
Makes sense. Changed it back!